source/acquaman/BioXAS/BioXASXASScanActionController.cpp

```diff
@@ -68,8 +70,9 @@ AMAction3* BioXASXASScanActionController::createInitializationActions()
 	// Initialize the scaler.
 
 	AMSequentialListAction3 *scalerInitialization = 0;
-	CLSSIS3820Scaler *scaler = CLSBeamline::clsBeamline()->scaler();
+//	CLSSIS3820Scaler *scaler = CLSBeamline::clsBeamline()->scaler();
```

Comment:
I think I would prefer to keep them for now, until initializing the Zebra has been implemented.